genome_designer/genome_finish/graph_contig_placement.py

```diff
@@ -311,27 +477,50 @@ def contig_neighbors(vert):
                                 enter_ref))
                         break
                 visited.append(exit_contig)
-                queue.extend([n for n in contig_neighbors(exit_contig)
-                        if n not in visited])
+
+                extend_items = []
+                for n in contig_neighbors(exit_contig):
+                    if n not in visited:
+                        extend_items.append(n)
+
+                queue.extend(extend_items)
 
     return iv_list
 
 
 def translocation_walk(G):
-    ref_seq_uid = G.ref_intervals.vertices[0].seq_uid
+
```

Comment:
Same here. Beyond describing this for the supplement of the Millstone paper / readthedocs, might be nice to have a comment here or in this file summarizing how all this works.